src/components/common/AlertBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div
+    :class="`border ${bgColor(
+      alertType
+    )} shadow rounded-lg fixed top-3 right-3 left-3 sm:left-auto sm:max-w-sm sm:w-full p-4 flex z-50`"
+  >
+    <div>
+      <!-- Heroicon name: solid/check-circle -->
+      <svg
```

Comment:
The check icon will be the same regardless of the alert type. The icon should be different depending on the alert type (like having a 'X' mark or errors).